goethe/corpora.py

```diff
@@ -29,7 +29,14 @@ def tokens(self):
         """
         path = self.file_path(use_tokens=True)
         with open(path) as f:
-            yield from (l.strip().split() for l in f)
+            tokens = [line.strip().split() for line in f]
```

Comment:
This line will load the entire file into memory. Use `tokens = (line.strip().split() for line in f)` instead.